samples/vsphere/vcenter/setup/setup_cli.py

```diff
@@ -103,4 +104,8 @@ def build_arg_parser():
                         action='store',
                         help='NFS Server IP to setup datastore for samples run.'
                              'If not passed as argument, update testbed.py file')
+    parser.add_argument('-epassword', '--esxpassword',
```

Comment:
@pgbidkar do you want to move this up next to e2? Keep a space in between just to be consistent. Also it seems the short version of the command is not that short. How about -ep, or perhaps we can just use the full command?